test/common/tls/integration/ssl_integration_test.cc

```diff
@@ -765,13 +765,13 @@ TEST_P(SslCertficateIntegrationTest, ServerEcdsaClientRsaOnlyWithAccessLog) {
 
   auto log_result = waitForAccessLog(listener_access_log_name_);
   if (tls_version_ == envoy::extensions::transport_sockets::tls::v3::TlsParameters::TLSv1_3) {
-    EXPECT_THAT(log_result,
-                StartsWith("DOWNSTREAM_TRANSPORT_FAILURE_REASON=TLS_error:|268435709:SSL_routines:"
-                           "OPENSSL_internal:NO_COMMON_SIGNATURE_ALGORITHMS:TLS_error_end"));
+    EXPECT_EQ(log_result,
+              "DOWNSTREAM_TRANSPORT_FAILURE_REASON=TLS_error:|268435709:SSL_routines:"
+              "OPENSSL_internal:NO_COMMON_SIGNATURE_ALGORITHMS:TLS_error_end FILTER_CHAIN_NAME=-");
   } else {
-    EXPECT_THAT(log_result,
-                StartsWith("DOWNSTREAM_TRANSPORT_FAILURE_REASON=TLS_error:|268435640:"
-                           "SSL_routines:OPENSSL_internal:NO_SHARED_CIPHER:TLS_error_end"));
```

Comment:
Is previous test ending with 2 TLS_error_ends here?

Reply:
Yes, the full log_result text here (in `main`) is
```
"DOWNSTREAM_TRANSPORT_FAILURE_REASON=TLS_error:|268435640:SSL_routines:OPENSSL_internal:NO_SHARED_CIPHER:TLS_error_end:TLS_error_end FILTER_CHAIN_NAME=-"
```
but it passed since it was only checking the prefix.